src/keymap.cpp

```diff
@@ -169,6 +169,13 @@ static const std::vector<OpDesc> opdescs = {
 		translatable("Toggle source view"),
 		KM_ARTICLE
 	},
+	{
+		OP_MARKITEMREAD,
+		"mark-article-read",
+		"y",
+		translatable("Mark read status for article"),
```

Comment:
Almost forgot, but next to this built-in help text, all commands are also documented in our manpages/HTML-doc (via `doc/keycmds.dsv`)